src/main/protobuf/v1/catalog-item.proto

```diff
@@ -47,8 +47,11 @@ message CatalogDescriptorV1 {
   required int32 dailyCallsTotal = 10;
   optional AgreementApprovalPolicyV1 agreementApprovalPolicy = 11;
   optional int64 createdAt = 12;
-  optional int64 activatedAt = 13;
-  repeated string serverUrls = 14;
+  repeated string serverUrls = 13;
+  optional int64 publishedAt = 14;
+  optional int64 suspendedAt = 15;
```

Comment:
STOP! you can't swap fields in protobuf, this is not retrocompatible!

`serverUrls` should absolutely remain the field 14, and `activatedAt` can't be deleted but rather renamed!